src/v/kafka/server/handlers/produce.cc

```diff
@@ -221,6 +222,13 @@ static ss::future<produce_response::partition> produce_topic_partition(
             acks,
             num_records);
       });
+    return f.then([&octx, start](produce_response::partition p) {
+        if (p.error_code == error_code::none) {
+            auto dur = std::chrono::high_resolution_clock::now() - start;
```

Comment:
why not lowres ? high res is expensive tho. also why not a seastar type which already gets updated by the poller and does the tick every 10ms. folly does something similar too.

Reply:
can you characterize expensive? without it we deal with 20ms spread and to handle that we'll need to spend some time tuning.

Reply:
switch to chrono::steady_clock. should be cheaper than wall clock time presumably. not as cheap as lowres. but seems to be used a lot in seastar/scylla.